src/kernels/jupyter/finder/remoteKernelFinder.unit.test.ts

```diff
@@ -175,21 +175,22 @@ suite(`Remote Kernel Finder`, () => {
         disposables.push(kernelsChanged);
         kernelRankHelper = new KernelRankingHelper(preferredRemoteKernelIdProvider);
 
-        remoteKernelFinder = new RemoteKernelFinder(
+        remoteKernelFinder = new UniversalRemoteKernelFinder(
+            'currentremote',
+            'Local Kernels',
```

Comment:
This doesn't look correct here, plus the string should be localized if it's a display name.

Reply:
@IanMatthewHuff it's a test so I didn't try to make it localized.

Reply:
@rebornix was reading too fast. You are totally right :)